GATEKEEPERS.md

```diff
@@ -0,0 +1,27 @@
+# rust-vmm gate keeping
+
+Together with repositories code owners, the gate keepers team main
+role is to manage rust-vmm contributions. In particular, gate keepers will make
+sure that PRs get addressed, reviewed and eventually merged in a reasonable
+time frame.
+
+For that purpose, gate keepers may get assigned as PR reviewers when a rust-vmm
+repository is lacking code owners, or when a previously assigned reviewer is not
+able to review a pending PR.
+
+Gate keepers have write access to all rust-vmm repositories and can merge PRs
+whenever they pass all CI tests and have received at least 2 reviewer approvals.
+
+## Gate keepers
+
+* [Andreea Florescu](https://github.com/andreeaflorescu)
+* [Alexandru Agache](https://github.com/alexandruag)
+* [Alexandra Iordache](https://github.com/aghecenco)
+* [Jiang Liu](https://github.com/jiangliu)
+* [Paolo Bonzini](https://github.com/bonzini)
+* [Zach Reizner](https://github.com/zachreizner)
+* [Jenny Mankin](https://github.com/jennymankin)
+* [Sebastien Boeuf](https://github.com/sboeuf)
+* [Rob Bradford](https://github.com/rbradford)
+* [Chao Peng](https://github.com/chao-p)
+* [Samuel Ortiz](https://github.com/sameo)
```

Comment:
Should this list be in alphabetical order instead?

Reply:
Fixed.